### PR TITLE
Create new page

### PR DESCRIPTION
**Pull request by rgholway**
In our fetch mock we weren't able to test for the results on the show page.

### CODE REVIEW

**Review by CassandraNewell — DISMISSED** [review]
All controllers and models should have corresponding tests

**Comment by rgholway** [comment]
Should be good now

File: app/controllers/api/v1/reviews_controller.rb [hunk 1]
```diff
@@ -1,5 +1,19 @@
 class Api::V1::ReviewsController < ApplicationController
-  def index
-    render json: Game.find(params[:game_id]).reviews.order("created_at DESC"), adapter: :json
+protect_from_forgery unless: -> { request.format.json? }
+  def new
+  end
+
+  def create
+    # data = JSON.parse(request.body.read)
```
Comment (CassandraNewell) [inline app/controllers/api/v1/reviews_controller.rb]: Please remove this and any other commented-out code

File: config/routes.rb [hunk 2]
```diff
@@ -1,11 +1,10 @@
...
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
-  namespace :api do
-    namespace :v1 do
-      resources :games, only: [:index, :show, :create] do
-        resources :reviews, only: :index
-      end
+    namespace :api do
+      namespace :v1 do
+        resources :games, only: [:index, :create, :show] do
+          resources :reviews, only: [:new, :create]
```
Comment (CassandraNewell) [inline config/routes.rb]: I notice that you're no longer allowing a `index` action for reviews. Please check that this change is intentional.

**Review by CassandraNewell — DISMISSED** [review]
Please review these comments. After you remove the commented-out line and verify that you don't want an index route for your reviews, feel free to *merge this yourself* :)

**Pull request merged**